bom/openhab-addons/pom.xml

```diff
@@ -1,1329 +1,1334 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
Seems that you re-formatted the whole file. It might be sufficient to run `mvn spotless:apply`.